elasticsearch.md

```diff
@@ -1274,7 +1274,12 @@ readonlyrest:
     - name: "ReadonlyREST Enterprise instance #1"
       ror_kbn_auth:
         name: "kbn1"
-        roles: ["SAML_GRP_1"]
+        roles: ["SAML_GRP_1", "SAML_GRP_2"] # <-  use this rule when a user should have one of the listed roles
```

Comment:
in other authorization rules, we have the `groups` field. In JWT-based rules, we maintain `roles` for the backward comparibility, but for sake of consistency, `groups` was introduced here some time ago. So, let's have `groups` in examples. And let's don't use the `roles` word when we write about groups.

Reply:
`use this rule when a user should` ... `roles|groups` is a field or property of the `ror_kbn_auth` rule